README.md

```diff
@@ -42,6 +43,9 @@ python cs_upload.py --trace-file foo.cst.gz ...
   you want, navigate to `/organization/profiles` and then select an appropriate profile.
   In `/organization/profiles/211` or `/organization/profiles/211/detail`, the profile
   number is 211.
+- `--slot-name` (optional): name of the (existing or not) slot the trace should be uploaded
+  to. Without it, the API uses the first existing compatible slot it finds. If none exists 
+  yet, it will chooses the trace type as a new slot name.
```

Comment:
**suggestion (non-blocking)**: it will use the trace type as the name of the new slot?

Reply:
Way better, thanks!